smithy-gradle-plugin/src/it/java/software/amazon/smithy/gradle/SourceProjectionTest.java

```diff
@@ -26,7 +26,7 @@ public void testSourceProjection() {
             BuildResult result = GradleRunner.create()
                     .forwardOutput()
                     .withProjectDir(buildDir)
-                    .withArguments("clean", "build", "--stacktrace", "--debug")
```

Comment:
Removed `--debug` because it was causing too much noise and making the test failure outputs unnecessarily large.